test/workbox-routing/node/Router.mjs

```diff
@@ -0,0 +1,302 @@
+import clearRequire from 'clear-require';
+import sinon from 'sinon';
+import {expect} from 'chai';
+
+import Route from '../../../packages/workbox-routing/lib/Route.mjs';
+import Router from '../../../packages/workbox-routing/lib/Router.mjs';
+import expectError from '../../../infra/utils/expectError';
+import {_private} from '../../../packages/workbox-core/index.mjs';
+
+describe(`[workbox-routing] Router`, function() {
+  const sandbox = sinon.sandbox.create();
+  const {logger} = _private;
+  const MATCH = () => {};
+  const HANDLER = () => {};
+  const METHOD = 'POST';
+  const EXPECTED_RESPONSE_BODY = 'test body';
+
+  beforeEach(async function() {
+    clearRequire.all();
+
+    // Run this in the `beforeEach` hook as well as the afterEach hook due to
+    // a mocha bug where `afterEach` hooks aren't run for skipped tests.
+    // https://github.com/mochajs/mocha/issues/2546
+    sandbox.restore();
+    // Prevent logs in the mocha output.
+    sandbox.stub(logger, 'warn');
+    sandbox.stub(logger, 'log');
+  });
+
+  afterEach(function() {
```

Comment:
Change this to `after()` . I realised in some of my recent tests afterEach didn't run after the test suite had finished meaning stubs were lingering. and afterEach isn't needed if beforeEach alway cleans stubs.